src/app/app.component.ts

```diff
@@ -36,14 +29,10 @@ export class AppComponent implements OnInit {
   /**
   * To get the form from the backend
   */
-  userRoles() {
-    //  const rolesdata =  this.usersService.getCurrentUserRoles();
-    this.usersService.getUserRoles().subscribe(data => {
-      this.roles = data['result'];
-      localStorage.setItem('user_role', this.roles.roles[0]);
-    }, error => {
-
-    });
+ async userRoles() {
+     const rolesdata =  await this.usersService.getUserRoles();
+     console.log('apppcomponent', rolesdata);
+     localStorage.setItem('user_role', rolesdata['result'].roles);
```

Comment:
are we using local storage now??

Reply:
just storing i am not using any where i will remove

Reply:
for token and user details i am storing